### ISSUE TITLE
Does not compile

### ISSUE THREAD

**Issue by DubiDee** [algernon-A/MorePathUnits#4]
Not able to compile this mod

Changing 
`public class OptionsPanel : UIPanel`
in Code/Settings/OptionsPanel.cs to OptionsPanelBase and changing the internal method to protected setup allows it to compile and run

**Comment by algernon-A** [comment]
Correct, I haven't yet updated this one to the new commons standard.  I'm slowly going through all my mods and updating them, and this one should be done by the end of the month.

**Comment by DubiDee** [comment]
Could you post releases on github as well? I have to compile mods manually ever since the steam crackdown
